pom.xml

```diff
@@ -100,10 +100,10 @@
         <!-- Kafka + Confluent versions-->
         <scala.version>2.11</scala.version>
         <scala-library.version>${scala.version}.11</scala-library.version>
-        <kafka.confluent.version>0.11.0.1-cp1</kafka.confluent.version>
+        <kafka.confluent.version>1.1.1-cp2</kafka.confluent.version>
```

Comment:
Cross-check other dependencies.  https://github.com/apache/kafka/blob/1.1/gradle/dependencies.gradle

- RocksDB
- Snappy
- Jackson
- Jetty

Reply:
👍

Reply:
Upgraded all other related libraries except `Jetty` because we were already on a higher version.